src/java.naming/share/classes/com/sun/jndi/ldap/LdapDnsProviderService.java

```diff
@@ -50,25 +47,10 @@ final class LdapDnsProviderService {
     /**
      * Creates a new instance of LdapDnsProviderService
      */
-    @SuppressWarnings("removal")
     private LdapDnsProviderService() {
-        SecurityManager sm = System.getSecurityManager();
-        if (sm == null) {
-            providers = ServiceLoader.load(
-                    LdapDnsProvider.class,
-                    ClassLoader.getSystemClassLoader());
-        } else {
-            final PrivilegedAction<ServiceLoader<LdapDnsProvider>> pa =
-                    () -> ServiceLoader.load(
-                            LdapDnsProvider.class,
-                            ClassLoader.getSystemClassLoader());
-
-            providers = AccessController.doPrivileged(
-                pa,
-                null,
-                new RuntimePermission("ldapDnsProvider"),
-                SecurityConstants.GET_CLASSLOADER_PERMISSION);
```

Comment:
Can you check if SecurityConstants.GET_CLASSLOADER_PERMISSION be removed now, I think this is the last usage.